spec/factories/games.rb

```diff
@@ -23,12 +23,11 @@
       end
 
       after(:create) do |game, evaluator|
-        aggregate_list = create(:aggregate_shopping_list, game: game)
```

Comment:
This didn't really need to be created explicitly since it'll be automatically created when other lists are created.